libp2p/Host.cpp

```diff
@@ -396,6 +396,19 @@ void Host::startPeerSession(Public const& _id, RLP const& _hello,
                   << _s->remoteEndpoint();
 }
 
+/// Get session by id
+shared_ptr<SessionFace> Host::peerSession(NodeID const& _id) const
+{
+    RecursiveGuard l(x_sessions);
+    if (m_sessions.count(_id))
+    {
+        auto const s = m_sessions[_id].lock();
+        if (s && s->isConnected())
+            return s;
+    }
+    return{};
```

Comment:
Formatting.

Reply:
Thanks, still need to get git clang-format up and running again after a reinstall, will reformat manually for the time-being.